plugins/kafka-backend/src/config/ClusterReader.ts

```diff
@@ -15,23 +15,21 @@
  */
 
 import { Config } from '@backstage/config';
-import { ConnectionOptions } from 'tls';
-import { ClusterDetails } from '../types/types';
+import { ClusterDetails, SslConfig, SaslConfig } from '../types/types';
 
 export function getClusterDetails(config: Config[]): ClusterDetails[] {
   return config.map(clusterConfig => {
     const clusterDetails = {
       name: clusterConfig.getString('name'),
       brokers: clusterConfig.getStringArray('brokers'),
     };
-    const sslConfig = clusterConfig.getOptional('kafka.ssl');
```

Comment:
This nested cluster key `kafka.ssl` seems to have been incorrect. It doesn't match what is defined in the `config.d.ts` file. Not sure if this was a mistake or not. @nirga - Can you confirm this?